src/Renci.SshNet/SftpClient.cs

```diff
@@ -2238,7 +2237,9 @@ private IEnumerable<FileInfo> InternalSynchronizeDirectories(string sourcePath,
                         var remoteFileName = string.Format(CultureInfo.InvariantCulture, @"{0}/{1}", destinationPath, localFile.Name);
                         try
                         {
+#pragma warning disable CA2000 // Dispose objects before losing scope
                             using (var file = File.OpenRead(localFile.FullName))
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
What is the suppression referring to?

Reply:
It's a bug in the roslyn analyzer. Clarified this in latest commit.
I wanted to submit a bug against the roslyn analyzer, but I can't seem to create a small repro for this issue.